.github/workflows/default.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           node-version: 16
 
       - name: Checkout Squid sources
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Please move these checkout version changes into a dedicated PR.

Reply:
Done in PR #1786